DatadogInternal/Sources/NetworkInstrumentation/FirstPartyHosts.swift

```diff
@@ -8,7 +8,7 @@ import Foundation
 
 /// A struct that represents a dictionary of host names and tracing header types.
 public struct FirstPartyHosts: Equatable {
-    internal var hostsWithTracingHeaderTypes: [String: Set<TracingHeaderType>]
+    public var hostsWithTracingHeaderTypes: [String: Set<TracingHeaderType>]
```

Comment:
q/ Is this intentional? Where does it come from?

Reply:
needed in previous interation, not anymore reverted.